### PR TITLE
Propagate kwargs through `keras.ops.isclose`

### PR DESCRIPTION
**Pull request by LarsKue**
This allows passing `atol` and `rtol`. Should work under all backends.

I haven't found my way around the test suite yet. Is there a good place to implement a unit test for this?

### CODE REVIEW

**Comment by fchollet** [comment]
Thanks for the PR. If we want to support atol and rtol, we should explicitly add them to the `call()` signature of the operation class and to the `isclose` function signature, rather than using `kwargs`.

**Comment by LarsKue** [comment]
@fchollet Thanks for the suggestion. I changed this now.

File: keras/src/ops/numpy.py [hunk 1]
```diff
@@ -2801,8 +2801,8 @@ def imag(x):
 
 
 class Isclose(Operation):
-    def call(self, x1, x2):
-        return backend.numpy.isclose(x1, x2)
+    def call(self, x1, x2, rtol=1e-5, atol=1e-8, equal_nan=False):
```
Comment (fchollet) [inline keras/src/ops/numpy.py:2804]: If you change the signature in `keras.ops`, you need to change the backend function for all 4 backends, not just numpy.

Comment (LarsKue) [reply keras/src/ops/numpy.py:2804]: Of course, silly mistake. I am still finding my way into the library. Fixed now!

**Comment by LarsKue** [comment]
I adjusted the tensorflow version of `keras.ops.isclose` to use `tensorflow.experimental.numpy.isclose`. Even though this api is within experimental, I assume it is stable enough to use as-is. Happy to revert this, though, if there is a good reason to.

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**